chart/templates/controller/deployment.yaml

```diff
@@ -49,6 +49,13 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: JOBSUBMISSION_GRAPHQLENDPOINT
```

Comment:
We moved job submission graph endpoint and secret here so we have only one single source of the secret so we can random a secret if user didn't provided.

We modified primehub-controller, it will load env variables and then the config yaml. https://github.com/InfuseAI/primehub-controller/commit/69830f1703c9b62af3dbf0def608480f66f37707

cc @popcornylu